TEST.md

```diff
@@ -162,3 +162,16 @@ To audit a page's WCAG and Section 508 accessibility:
   4. To test a page aside from the homepage, add the `--u=<path_to_test>` flag.
      For example, `gulp test:a11y --u=contact-us`
      or `gulp test:a11y --u=the-bureau/bureau-structure/`.
+
+# Source code linting
+
+The default test task includes linting of the JavaScript source, build,
+and test files.
+Use the `gulp lint` command from the command-line to run the ESLint linter,
+which defines a number of [rules](http://eslint.org/docs/rules/).
+There are a number of options to the command:
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2237/commits/7a1d73933d8530fe6e68733647ae37dc04a1d539